service/src/main/java/com/fecklessweasel/service/CourseServlet.java

```diff
@@ -32,6 +33,27 @@ protected void doPost(final HttpServletRequest request,
                           final HttpServletResponse response)
             throws ServletException, IOException {
 
+        String methodType = request.getParameter("method");
+        switch (methodType) {
+            case "handleCourseCreate":
+                handleCourseCreate(request, response);
+                break;
+            default:
+                // do nothing
+        }
+    }
+
+    /**
+     * Handles post requests to create a new class.
+     * @param request contains parameters course and dept
+     * @param response directs to course/index.jsp
+     * @throws ServletException When any handled error happens.
+     * @throws IOException Server was unable to process the request.
+     */
+    protected void handleCourseCreate(final HttpServletRequest request,
```

Comment:
This may seem unnecessary - and probably is, but I originally planned to use a form to post to the course servlet and realized this is the easiest way to post to the same Servlet, but use different methods.